### ISSUE TITLE
Can‘t build sample with 1.200.0

### ISSUE THREAD

**Issue by picofpv** [googlevr/gvr-android-sdk#632]
gvrsdk: com.google.vr:sdk-audio:1.200.0
Can‘t build 

ERROR: Failed to resolve: com.google.vr:sdk-base:1.200.0
Show in Project Structure dialog
Affected Modules: sdk-controllerclient, sdk-hellovr, sdk-video360, sdk-videoplayer

ERROR: Failed to resolve: com.google.vr:sdk-audio:1.200.0
Show in Project Structure dialog
Affected Modules: sdk-videoplayer

————————————————————————————
And I can't find VR in maven.google.com, only AR

![novr](https://user-images.githubusercontent.com/47655597/55821885-137d1380-5b31-11e9-99de-b8f8f3d1ccec.jpg)

**Comment by zhiyuhuo** [comment]
me too. google vr really got a problem.

**Comment by tillkrempel** [comment]
please deploy 1.200.0

**Comment by JohnCWakley** [comment]
Same here:
ERROR: Failed to resolve: com.google.vr:sdk-base:1.200.0

**Comment by dpr10** [comment]
Just change the version to 1.190 and all works just fine

**Comment by JohnCWakley** [comment]
> Just change the version to 1.190 and all works just fine

Do you mean, download version 1.190, or change the version within the configs to 1.190?

**Comment by picofpv** [comment]
Downgrade dependencies to 1.190 ，
Most of the examples work fine，but **ndk-hellovrbeta** logcat ERROR

2019-04-24 23:08:04.860 20452-20452/com.google.vr.ndk.samples.hellovrbeta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.google.vr.ndk.samples.hellovrbeta, PID: 20452
    java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "**gvr_beta_see_through_config_create**" referenced by "/data/app/com.google.vr.ndk.samples.hellovrbeta-ANr4SJWmtgJKes6ajdiQwQ==/lib/x86/libhellovrbeta_jni.so"...

**Comment by dpr10** [comment]
> > Just change the version to 1.190 and all works just fine
> 
> Do you mean, download version 1.190, or change the version within the configs to 1.190?

Just change the dependencies in gradle configuration to 1.190, thats works for me

**Comment by dpr10** [comment]
> Downgrade dependencies to 1.190 ，
> Most of the examples work fine，but **ndk-hellovrbeta** logcat ERROR
> 
> 2019-04-24 23:08:04.860 20452-20452/com.google.vr.ndk.samples.hellovrbeta E/AndroidRuntime: FATAL EXCEPTION: main
> Process: com.google.vr.ndk.samples.hellovrbeta, PID: 20452
> java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "**gvr_beta_see_through_config_create**" referenced by "/data/app/com.google.vr.ndk.samples.hellovrbeta-ANr4SJWmtgJKes6ajdiQwQ==/lib/x86/libhellovrbeta_jni.so"...

I don't run the NDK samples, but checking again, this could be an imcompatibilities with the SDK version 1.190, if you check the log on the **ndk-hellovrbeta**, the jni source code has been update to versión 1.200, maybe this could be the source of the problem

**Comment by tillkrempel** [comment]
The problem seems to be that the 1.200 library artifact has not been published in the google maven repo. Unfortunately IMHO this can only be fixed by someone with the credentials to do so. Downgrading to a previous version (as suggested above) will fix the issue that the project cannot be built, but new features of 1.200, like enabling see-through mode will not work with the old artifact.

It should be possible to get the artifact from a local maven repo, or from the plain file system by adding a repository to the gradle config, but I was not yet able to make it work like that.

**Comment by dustyfooted** [comment]
> > > Just change the version to 1.190 and all works just fine
> > 
> > 
> > Do you mean, download version 1.190, or change the version within the configs to 1.190?
> 
> Just change the dependencies in gradle configuration to 1.190, thats works for me

Please I am completely new. How do I change the dependency ?
I get ERROR: Failed to resolve: com.google.vr:sdk-base:1.200.0
First time running. I would appreciate it if you can direct me on how to solve it.

**Comment by dpr10** [comment]
Just change de version in `build.gradle` file, under dependencies section.

**Original**
 `
dependencies {
    implementation 'com.android.support:appcompat-v7:26.1.0'
    implementation 'com.google.vr:sdk-base:1.200.0'
}
`

**Modified**
`
dependencies {
    implementation 'com.android.support:appcompat-v7:26.1.0'
    implementation 'com.google.vr:sdk-base:1.190.0'
}
`

**Comment by bell-one** [comment]
You can use sdk-base-1.200.0.aar directly from your project.
to do this.
1. import aar to your project
2. add dependecies implementation project(**':sdk-base-1.200.0'**)
    bold one will be imported name of 'aar'

then you can use this without maven repos

**Comment by TheDeveloperGuy** [comment]
How nice would it be if Google tested their SDK on the latest version of the official developer tools for their operating system? *Every single time* one of these SDKs is released the samples fail to build.

**Comment by StrikeNP** [comment]
Just bumping that I am also having this issue.

**Comment by TheDeveloperGuy** [comment]
It's possible using the (manual) steps above to get the samples to build and deploy (either downgrade dependencies to 1.190 or use 1.200 directly from your project). The lack of support for GoogleVR is astounding. The only platform that can use the features of the 1.200 SDK (Mirage Solo) is not able to run executables that are built with the 1.200 SDK, it requests a later version of the Daydream support libraries and none is available. So, one can conclude that the 1.200 SDK examples have never actually been tested. :/

**Comment by StrikeNP** [comment]
> It's possible using the (manual) steps above to get the samples to build and deploy (either downgrade dependencies to 1.190 or use 1.200 directly from your project). The lack of support for GoogleVR is astounding. The only platform that can use the features of the 1.200 SDK (Mirage Solo) is not able to run executables that are built with the 1.200 SDK, it requests a later version of the Daydream support libraries and none is available. So, one can conclude that the 1.200 SDK examples have never actually been tested. :/

I tried the manual downgrade steps from above and was able to get some of the samples to work. Due to the nature of my project however and increased community support I simply switched to building in Unity, and ran into fewer (but not 0) issues.

**Comment by picofpv** [comment]
I am sad to see that Google VR can't get more official support. The technology of Google VR and Tango was once advanced, but now it can only follow other companies. Although GoogleVR programmers are very capable, the company's top management seems to have no clear vision for the future.

**Comment by Consti10** [comment]
Only thing that worked for me was downgrading to 1.190.0

**Comment by tillkrempel** [comment]
Yeah, but then you cannot use the see through mode. That's the main reason
I bought the Lenovo headset for... seriously, it has been months and still
noone at Google has time to deploy an artifact ?

Consti10 <notifications@github.com> schrieb am Do., 29. Aug. 2019, 16:47:
